spec/functions/number.md

```diff
@@ -159,40 +159,41 @@ together with the resolved options' values.
 
 The _function_ `:integer` performs selection as described in [Number Selection](#number-selection) below.
 
-#### The `:math` function
+#### The `:offset` function
 
-> [!IMPORTANT]
-> The _function_ `:math` has a status of **Draft**.
-> It is proposed for inclusion in a future release of this specification and is not Stable. 
-
-The _function_ `:math` is proposed as a _selector_ and _formatter_ for matching or formatting
-numeric values to which a mathematical operation has been applied.
+The _function_ `:offset` is a _selector_ and _formatter_ for matching or formatting
+numeric values to which an offset has been applied.
+The "offset" is a small integer adjustment of the _operand_'s value.
 
 > This function is useful for selection and formatting of values that
 > differ from the input value by a specified amount.
-> For example, it can be used in a message such as this:
+> For example, it can be used in a _message_ such as this:
 >
 > ```
 > .input {$like_count :integer}
-> .local $others_count = {$like_count :math subtract=1}
+> .local $others_count = {$like_count :offset subtract=1}
 > .match $like_count $others_count
 > 0 *   {{Your post has no likes.}}
 > 1 *   {{{$name} liked your post.}}
 > * one {{{$name} and {$others_count} other user liked your post.}}
 > * *   {{{$name} and {$others_count} other users liked your post.}}
 > ```
 
-##### Operands
+> [!NOTE]
+> The purpose of this _function_ is to supply compatibility with
+> ICU's `PluralFormat` and its `offset` feature, also found in ICU MessageFormat.
```

Comment:
```suggestion
> ICU's `PluralFormat` and its `offset` feature, also found in ICU MessageFormat.
> It is needed for migration, because it is commonly used for messages
> that need to convey the first few items of a list plus the count of the remainder,
> such as in the example above.
```

Reply:
I think we're inverting things a bit here. As I see it, the primary purpose of this function is the same as that of plural offset in MF1, to enable the expression of messages with plural variance that depends on an offset count. Supporting migration from MF1 arises from that; it ought not be presented as "the purpose" of the function as a whole.